README.md

```diff
@@ -0,0 +1,146 @@
+# backplane-tools
+
+backplane-tools offers an easy solution to install, remove, and upgrade a useful set of tools for interacting with OpenShift clusters, managed or otherwise.
+
+## Table of Contents
+<!-- toc -->
+- [Tools](#tools)
+- [FAQ - How do I...](#faq-how-do-i)
+  - [List available tools](#list-available-tools)
+  - [Install everything](#install-everything)
+  - [Install a specific thing](#install-a-specific-thing)
+  - [Upgrade everything](#upgrade-everything)
+  - [Upgrade a specific thing](#upgrade-a-specific-thing)
+  - [Remove everything](#remove-everything)
+  - [Remove a specific thing](#remove-a-specific-thing)
+- [Design](#design)
+  - [Directory Structure](#directory-structure)
+  - [Installing](#installing)
+  - [Upgrading](#upgrading)
+  - [Removing](#removing)
+<!-- tocstop -->
+
+## Tools
+The tools currently managed by this application include:
+
+* aws
+* backplane-cli
+* ocm
+* osdctl
+* rosa
```

Comment:
What about oc?
```suggestion
* rosa
* oc
```

Reply:
The oc logic isn't actually functional yet